src/main/java/com/desiato/puresynth/models/UserDTO.java

```diff
@@ -0,0 +1,3 @@
+package com.desiato.puresynth.models;
+
+public record UserDTO(Long id, String email) {}
```

Comment:
It's good to have a convetion for all request response DTOs. I suggest:
* `<Entityname>RequestDTO` for request objects
* `<Entityname>ResponseDTO` for request objects